packages/grafana-e2e/src/flows/importDashboards.ts

```diff
@@ -0,0 +1,26 @@
+import { importDashboard, Dashboard } from './importDashboard';
+import { e2e } from '../index';
+
+/**
+ * Smoke test several dashboard json files from a test directory
+ * and validate that all the panels in each import finish loading their queries
+ * @param dirPath the path to a directory which contains json files representing dashboards.
+ * A note that due this directory must be relative to `node_modules/@grafana/e2e/cypress/plugins`
+ * so an example path might be something like `../../../../../cypress/fixtures/testDashboards`
```

Comment:
You can try to use the [cypress config](https://docs.cypress.io/api/cypress-api/config#Syntax) `Cypress.config('integrationFolder')`. That will point to the folder containing the tests so paths can be relative to the test file being written.

Reply:
Neat! Looks like there's a freebie property already there that offers us what we need :D adding now.